### PR TITLE
Add catalog infra template for external dns used with gandi

### PR DESCRIPTION
**Pull request by fredleger**

### CODE REVIEW

**Comment by fredleger** [comment]
should fix rancher/rancher#8823

**Comment by janeczku** [comment]
Thanks @fredleger! Will merge after review.

File: infra-templates/gandi-dns/0/docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+gandidns:
+  image: rancher/external-dns:v0.6.4
```
Comment (janeczku) [inline infra-templates/gandi-dns/0/docker-compose.yml]: Please change to: `rancher/external-dns:v0.7.2`

File: infra-templates/gandi-dns/0/rancher-compose.yml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    - variable: "ROOT_DOMAIN"
+      label: "Root Domain"
+      description: "The domain name managed by Gandi."
+      type: "string"
+      required: true
+    - variable: "TTL"
+      label: "TTL"
+      description: "The resource record cache time to live (TTL), in seconds (minimum 300)"
+      type: "int"
+      default: 1500
```
Comment (janeczku) [inline infra-templates/gandi-dns/0/rancher-compose.yml]: Any reason you are not using the minimum TTL as default? 5 mins is already quite high for dynamic environments.

File: infra-templates/gandi-dns/0/rancher-compose.yml [hunk 3]
```diff
@@ -0,0 +1,41 @@
+.catalog:
```
Comment (janeczku) [inline infra-templates/gandi-dns/0/rancher-compose.yml]: Please add the `# notemplating` tag to the top of the file and `minimum_rancher_version: v1.6.0-rc1`. See https://github.com/rancher/community-catalog/blob/master/infra-templates/digitalocean-dns/3/rancher-compose.yml#L1

File: infra-templates/gandi-dns/config.yml [hunk 4]
```diff
@@ -0,0 +1,5 @@
+name: GANDI DNS
+description: |
+  Rancher External DNS service powered by Gandi.net
+version: v0.1.0
+category: External DNS
```
Comment (janeczku) [inline infra-templates/gandi-dns/config.yml]: Please add the label as shown here https://github.com/rancher/community-catalog/blob/master/infra-templates/digitalocean-dns/config.yml#L6-L7

**Comment by fredleger** [comment]
@janeczku : this should fix the points discussed together

**Comment by janeczku** [comment]
@fredleger thanks

**Comment by janeczku** [comment]
LGTM

**Pull request merged**